doc/source/cpp/README.md

```diff
@@ -0,0 +1,3 @@
+# Packaging a C++ Framework/Model for Seldon Core
+
+
```

Comment:
Should we add links here to the examples so that the documentation page doesn't look empty?

Reply:
Yeah my bad, this is the main documentation page I need to fill up for at least having a baseline!

Reply:
Added documentation